package.json

```diff
@@ -25,7 +26,7 @@
   "dependencies": {
     "chalk": "^2.4.2",
     "crypto-ld": "^3.7.0",
-    "did-veres-one": "^8.0.0",
+    "did-veres-one": "veres-one/did-veres-one#update-context",
```

Comment:
```suggestion
    "did-veres-one": "^9.0.0",
```

Reply:
did-veres-one@9 should now have the updated context, and Tobias's assertMethod feature.